cloudflare/resource_cloudflare_load_balancer.go

```diff
@@ -434,6 +611,158 @@ func resourceCloudflareLoadBalancerImport(d *schema.ResourceData, meta interface
 	return []*schema.ResourceData{d}, nil
 }
 
+func flattenRules(rules []*cloudflare.LoadBalancerRule) (interface{}, error) {
+
+	if len(rules) == 0 {
+		return nil, nil
+	}
+
+	bb, err := json.Marshal(rules)
+	if err != nil {
+		return nil, err
+	}
+
+	var rout []interface{}
+	if err := json.Unmarshal(bb, &rout); err != nil {
+		return nil, err
+	}
+
+	// terraform doesn't support maps of maps so lets
```

Comment:
generally, if you want a map of maps, you probably _really_ want a list of maps instead 😃